### PR TITLE
Porfolio Set Up Mobile First Pull Request

### PR DESCRIPTION
**Pull request by federicaulzurrun**
HTML file includes:
 . Header 
      1.    Nav   
 . Main
      1. h1
      2. p
      3. h6
      4. ul

CSS file includes:

- container-nav
- logo-name
- navlog
- container (img background)
- intro
- connect
- networks
- 

-

### CODE REVIEW

**Comment by mukhammadrizooff** [comment]
@federicaulzurrun Please fix linter issue 🙏🏼

**Comment by federicaulzurrun** [comment]
i fixed them, thank you! @mukhammadrizooff

File: style.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 24px;
+  padding-top: 10px;
+  gap: 12px;
+}
+
+.linkedin {
+  color: solid #505f79;
+}
```
Comment (mukhammadrizooff) [inline style.css]: - [ ] [REQUIRED] Good start! In the `header` section, give the right `color` value to the elements, it is a little too `light`, pay attention to the `position` of the main `heading` and `paragraph` and give the desired style, as shown in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=8kOjLtGM8zmLoMSW-0), and pay attention to the `color` of the `social icons` some of them are not in the required color. Finally, the `background image` is not available at the bottom of the figure web page, so please `style` it as shown in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=8kOjLtGM8zmLoMSW-0) 🙏🏼 🙏🏼 I believe you will do it easily and I wish you to approve in the next review 😃 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @federicaulzurrun, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mukhammadrizooff** [comment]
@federicaulzurrun Good luck 😃

File: style.css (hunk 1 above)
Comment (federicaulzurrun) [reply style.css]: @mukhammadrizooff in the logo and the linkedin logo i dont know what its happening cause the color its the one that is on figma, do you know what could be doing that?

File: style.css [hunk 2]
```diff
@@ -23,23 +26,37 @@
 }
 
 .container {
-  background-image: url(/img/background.png);
+  background-image: url(/img/background2.png);
+  width: auto;
+  height: auto;
+  margin: none;
+  margin-top: 40px;
+}
```
Comment (Tufahel) [inline style.css]: - [ ] Well done on implementing previous reviewer suggestions. 👍 But, there is still some issue. The background image is repeated twice in the same frame, so there are some there `extra shapes` coming that are not in the design guideline.
  - I suggest you use  ` background-repeat: no-repeat;` and `background-size: cover;` in the `container` class to fix this issue. Kindly consider this.
  - ***Your Design***
  image

File: style.css [hunk 3]
```diff
@@ -50,23 +67,25 @@
...
   gap: 12px;
 }
 
-.linkedin {
-  color: solid #505f79;
+.fa-linkedin-in {
+  color: #505f79;
+  width: 18px;
+  height: 20px;
 }
```
Comment (Tufahel) [inline style.css]: - [ ] Linkedin icon is still not fixed. Kindly use this [link](https://drive.google.com/file/d/1nrZf71ka2ymqqnItTz-wG4j7ziEUkSm6/view?usp=share_link) to download the png file and fix the issue as per previous reviewers suggestion.

  image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @federicaulzurrun  :wave: 

Well done on implementing some of the required changes. :+1:  
You still need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing
✔️ Use GitHub Flow
✔️ Meaningful commit messages.

## Required Changes ♻️

_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amon-cofie — APPROVED** [review]
# Approved 🟢 
Hi @federicaulzurrun , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and great job on implementing the changes suggested by previous reviewers! 🎉 



## Highlights

🚀 Passing linters
🚀 Desriptive pull request
🚀 Github flow 

## Optional suggestions
[OPTIONAL] In future projects, please aspire to make your commit messages more imperative. Example, instead of ```Adding flexbox to the project```, you should say ```Add flexbox to the project```

[OPTIONAL] Awesome work done on this project. However, there seems to be a problem with your deployment to github pages. Please check to ensure that you are deploying this from the right branch perhaps. 

image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by federicaulzurrun** [comment]
Thank you so much!!!!! @amon-cofie

**Pull request merged**